### PR TITLE
Add example using within expression

### PR DESCRIPTION
**Pull request by tobrun**
ezgif com-video-to-gif (12)

### CODE REVIEW

**Comment by alexshalamov** [comment]
@tobrun would be cool if bufferLineStringGeometry is visualized, maybe worth adding semitransparent line layer that will draw 'within' geometry?

File: MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/turf/WithinExpressionActivity.kt [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+      )
+    )
+
+    // Convert to a LineString
+    val routeLineString = LineString.fromLngLats(coordinates)
+
+    // Create buffer around linestring
+    val bufferedRouteGeometry = bufferLineStringGeometry(routeLineString)
+
+    // Setup style with custom layers,
```
Comment (chloekraw) [inline MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/turf/WithinExpressionActivity.kt]: Why does this say "custom layers" here? Seems a bit confusing with the `MGLOpenGLStyleLayer` custom layers

Comment (tobrun) [reply MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/turf/WithinExpressionActivity.kt]: custom layers on top mapbox streets, I will change the naming ;)

**Comment by tobrun** [comment]
ezgif com-crop

**Pull request merged**